iroha-cli/interactive/impl/interactive_transaction_cli.cpp

```diff
@@ -333,7 +333,7 @@ namespace iroha_cli {
           parser::parseValue<boost::multiprecision::uint256_t>(params[2]);
       auto precision = parser::parseValue<uint32_t>(params[3]);
       if (not val_int or not precision) {
-        std::cout << "Wrong format for amount" << std::endl;
+        std::cout << "Wrong format for amount or precision should be greater than 0" << std::endl;
```

Comment:
These are 2 separate cases, so I suggest having two separate checks with different messages, one for amount and one for precision.

Also, that line is too long, our guideline is 80 characters per line. We use clang-format to keep code style consistent.

Reply:
Can we change our code style to 100 chars? I've looked at many of the files, and cutting things off at 80 hurts readability.

Reply:
The issue here is that on small screens it will be hard to read code longer than 80 chars. Also a lot of tools, including github were designed with 80 characters in mind. As you can see above std::cout line is about 94 characters long and already it does not fit into code window and we have to scroll right to see the rest of the line.